src/setters/price.ts

```diff
@@ -52,6 +54,12 @@ export const setGvtPrice = (): void => {
     price.save();
 }
 
+export const set3CrvPrice = (): void => {
+    let price = initPrice();
+    price.threeCrv = get3CrvVirtualPrice();
+    price.save();
+}
+
```

Comment:
In file `price.ts` there are all the price setters, so please move the code from `src/utils/threePool.ts` into function `set3CrvPrice()` and remove file `src/utils/threePool.ts`.
For accessing this price, we just do `price.threeCrv` instead of `get3CrvVirtualPrice()`.